src/components/calcite-color-picker/calcite-color-picker.scss

```diff
@@ -0,0 +1,278 @@
+$font-weight--medium: 500;
+$section-label-bottom-padding: 2px;
+$inline-button-size: 24px;
+$color-mode-selected-border-height: 3px;
+$channel-input-offset: 30%;
+
+:host {
+  border: 1px solid var(--calcite-ui-border-1);
+  display: inline-block;
+  background-color: var(--calcite-ui-foreground-1);
+}
+
+:host([dir="rtl"]) {
+  .channel-input {
+    padding: 0 $channel-input-offset 0 0;
+  }
+}
+
+:host([scale="s"]) {
+  width: 170px;
+
+  .header,
+  .channel-label {
+    font-size: 12px;
+  }
+
+  .channel {
+    width: 48px;
+    margin-right: 3px;
+  }
+}
+
+:host([scale="m"]) {
+  width: 240px;
+}
+
+:host([scale="l"]) {
+  width: 370px;
+
+  .control-section {
+    flex-wrap: nowrap;
+  }
+
+  .color-hex-options {
+    flex-basis: 33%;
+    flex-grow: 0;
+  }
+
+  .color-mode-container {
+    margin-top: 0;
+    flex-grow: 0.5;
+  }
+}
+
+.color-field-and-slider {
+  margin-bottom: -20px;
+
+  &--interactive {
+    cursor: pointer;
+  }
+}
+
+.control-section {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+}
+
+.section {
+  padding: 16px 10px 0;
+
+  &:last-child {
+    padding-bottom: 16px;
+  }
+}
+
+.control-section {
+  display: flex;
+  justify-content: space-between;
+}
+
+.color-hex-options {
+  flex-grow: 1;
+}
+
+.section--split {
+  margin-top: 16px;
+  flex-grow: 1;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  color: var(--calcite-ui-text-3);
+  font-weight: $font-weight--medium;
+  font-size: 14px;
+
+  &.header--underlined {
+    border-bottom: 1px solid var(--calcite-ui-border-3);
+  }
+}
+
+.color-mode,
+.header--hex {
+  @include focus-style-base();
+
+  padding-bottom: $section-label-bottom-padding;
+
+  &:focus {
+    @include focus-style-outset();
+  }
+}
+
+.header--hex {
+  padding-bottom: $section-label-bottom-padding + $color-mode-selected-border-height - 1px;
+}
+
+.control {
+  margin-top: 8px;
+}
+
+.color-mode {
+  cursor: pointer;
+  margin-bottom: -1px;
+  border-bottom: $color-mode-selected-border-height solid transparent;
+  transition: all 0.15s ease-in-out;
+
+  &:hover:not(.color-mode-selected) {
```

Comment:
Does `calcite-tab` itself here not work?

Reply:
I'll do this [in a follow up issue](https://github.com/Esri/calcite-components/pull/592#issuecomment-662789967) (using `calcite-tab`, `calcite-button` and `calcite-input`).

Reply:
Rad! - yeah I just wired up the calcite-inputs to test with `scale=s` and `prefix-text` and it works with just a couple of CSS updates :) https://jmp.sh/PFzQ1KV

Reply:
That's awesome! Almost there! 🎉